www/src/pages/en/usage/drizzle.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Drizzle
+description: Usage of Drizzle
+layout: ../../../layouts/docs.astro
+lang: en
+---
+
+Drizzle is an ORM for TypeScript, that allows you to define your database schema and models in a `schema.ts` file, and then generate a type-safe client that can be used to interact with your database from your backend.
+
+Drizzle is three major components
+
+- `drizzle-orm`: the ORM itself. Installed as a dependency.
+- `drizzle-kit`: a CLI app used for managing migrations. Installed as a dependency.
+- `drizzle-studio`: a GUI for managing you database. Installed as part of drizzle-kit.
+
+## Drizzle Config
+
+Located at `/drizzle.config.ts`. Here you can customize options like where your schema and migrations are located. Credentials provided here will be used by drizzle-kit for performing migrations. [Drizzle Documentation](https://orm.drizzle.team/docs/drizzle-config-file)
+
+## Drizzle Client
+
+Located at `src/server/db/index.ts`. The Drizzle Client is instantiated as a global variable and exported to be used in your API routes. We include the Drizzle Client in [Context](/en/usage/trpc#-serverapitrpcts) by default and recommend using this instead of importing it separately in each file. Credentials provided here will be used at runtime by drizzle-orm.
+
+## Drizzle Schema
+
+Located at `src/server/db/schema.ts`. This file is where you define your database schema and models. For those familiar with Prisma, the concepts remain the same but the syntax is different. The entire file is written in plain Typescript meaning it's a little more verbose. A major advantage however, is that schema types are applied immediately on save without the need to re-generate types after every change. [Drizzle Documentation](https://orm.drizzle.team/docs/sql-schema-declaration)
+
+### With NextAuth.js
+
+When you select NextAuth.js in combination with Drizzle, the schema file is generated and set up for you with the recommended values for the `User`, `Session`, `Account`, and `VerificationToken` models, as per the [NextAuth.js documentation](https://authjs.dev/getting-started/adapters/drizzle).
+
+## Default Database
+
+The default database is an SQLite database, which is great for development and quickly spinning up a proof-of-concept but is not recommended for production. You can change the database to use by changing the `provider` in the `datasource` block to either `postgresql` or `mysql`, and then updating the connection string within environment variables to point to your database.
```

Comment:
> a proof-of-concept but is not recommended for production

feel like we may need to re-evaluate this now with services like D1 and Turso 😅

Reply:
I rewrote this whole paragraph. It was copied verbatim from the Prisma section and it's not correct.